packages/wdio-devtools-service/src/index.js

```diff
@@ -5,12 +5,12 @@ import DevToolsDriver from './driver'
 import Auditor from './auditor'
 import TraceGatherer from './gatherer/trace'
 import DevtoolsGatherer from './gatherer/devtools'
-import { findCDPInterface, getCDPClient, isBrowserVersionLower } from './utils'
+import { findCDPInterface, getCDPClient, isBrowserSupported } from './utils'
 import { NETWORK_STATES, DEFAULT_NETWORK_THROTTLING_STATE } from './constants'
 
 const log = logger('@wdio/devtools-service')
-const UNSUPPORTED_ERROR_MESSAGE = 'The @wdio/devtools-service currently only supports Chrome version 63 and up'
 const TRACE_COMMANDS = ['click', 'navigateTo', 'url']
+const UNSUPPORTED_ERROR_MESSAGE = 'The @wdio/devtools-service currently only supports Chrome version 64 and up, and Chromium as the browserName!'
```

Comment:
`63` instead of `64`?
Based on your code:
```
SUPPORTED_BROWSERS_AND_MIN_VERSIONS = {
    'chrome': 63,
    'chromium' : 63,
    'google chrome': 63
}
```

Reply:
Good catch.